vulnerabilities/templates/index.html

```diff
@@ -1,14 +1,166 @@
-{% extends 'base.html' %}
+{% extends "base.html" %}
 
 {% block title %}
-<h1 class="title">
-Welcome to VulnerableCode!
-</h1>
+VCIO Home
 {% endblock %}
 
 {% block content %}
-<div class = "hero is-large has-text-centered">
-    <a class="button is-medium has-background-light has-text-link"  href="{% url 'vulnerability_search' %}"> Search Vulnerabilities</a>
-    <a class="button is-medium has-background-light has-text-link" href="{% url 'package_search' %}"> Search Packages</a>
-</div>
+
+{% include 'navbar.html' %}
+
+<section class="section01">
+
+    <div class="columns">
+        <div class="column">
+            <span class="tag01">Welcome to VulnerableCode<span class="nexb-orange">.</span>io.</span>
+        </div>
+    </div>
+
+    <article class='panel is-info'>
+        <div class='panel-heading py-2 is-size-6'>
+            Search for vulnerable packages
+            <div class="dropdown is-hoverable" style="font-weight: normal;">
+                <div class="dropdown-trigger" style="text-align: right;">
+                    <i class="fa fa-question-circle" style="font-size:20px; margin-left: 10px;"></i>
+                </div>
+                <div class="dropdown-menu" id="dropdown-menu4" role="menu" style="width: 600px;">
+                    <div class="dropdown-content" style="box-shadow: 0px 8px 16px 0px #808080;">
+                        <div class="dropdown-item" style="font-size: 14px;">
+                            <div style="margin: 0px 0px 0px 0px;">
+                                <div style="margin-top: 0px;">Search for currently known Vulnerabilities for a <a href="https://github.com/package-url/purl-spec" target="_blank">Package-URL</a> (aka <span class="inline-code01">purl</span>) or a Package <span class="inline-code01">name</span>.</div>
```

Comment:
Margin classes are available in the CSS framework for those style, for example:
`class="m-0"` for no margins at all or `class="mt-0"` for no margin on top.

Reply:
Done.